db-protocol/mysql/src/main/java/org/apache/shardingsphere/db/protocol/mysql/packet/binlog/row/column/value/time/MySQLFractionalSeconds.java

```diff
@@ -26,41 +27,29 @@
  */
 public final class MySQLFractionalSeconds {
     
-    private final int fraction;
+    @Getter
+    private final int nanos;
     
     private final int fractionalSecondsPrecision;
     
     public MySQLFractionalSeconds(final int columnMeta, final MySQLPacketPayload payload) {
         fractionalSecondsPrecision = columnMeta;
-        fraction = readFraction(payload);
+        nanos = readFraction(payload);
     }
     
     private int readFraction(final MySQLPacketPayload payload) {
```

Comment:
eg. the fractional seconds is `.77` means the precision is 2, equals 0.77 seconds, to nanos is `0.77 * 1000 * 1000 * 1000`, paylod.readInt1() will return 77
the fractional seconds is `.773` means the precision is 3, equals 0.773 seconds, to nanos is `0.773 * 1000 * 1000`, paylod.readUnsignedShort() will return 773